test/integration/targets/unarchive/tasks/test_missing_binaries.yml

```diff
@@ -39,7 +39,7 @@
       environment:
         PATH: "{{ ENV_PATH }}"
       vars:
-        ENV_PATH: "{{ lookup('env', 'PATH') | regex_replace(re, '') }}"
+        ENV_PATH: "{{ ansible_env['PATH']| regex_replace(re, '') }}"
```

Comment:
Why was this change needed?

Reply:
mostly for correctness, lookup queries 'controller' while ansible_env relates to the target, the action happens on the target, using testhost/localhost these are the same, but if we ever split, it won't be.